app/views/images/index.html.haml

```diff
@@ -0,0 +1,33 @@
+%div{:id => "addimages"}
+  %a(href='images/new') Add an image
+%br
+- if @images.any?
+  - @images.each do |image|
+    %div{:id => "images"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace